core/chains/evm/txmgr/eth_resender.go

```diff
@@ -112,11 +117,25 @@ func (er *EthResender) resendUnconfirmed() error {
 		if err != nil {
 			return errors.Wrap(err, "failed to FindEthTxAttemptsRequiringResend")
 		}
+		oldestAttempt, exists := findOldestUnconfirmedAttempt(attempts)
+		if exists {
+			// Wait at least 2 times the EthTxResendAfterThreshold to log critical with an unconfirmedTxAlertDelay
+			if (time.Since(oldestAttempt.CreatedAt) > er.config.EthTxResendAfterThreshold()*2) &&
+				time.Since(er.lastAlertTimestamps[k]) >= unconfirmedTxAlertDelay {
+				er.lastAlertTimestamps[k] = time.Now()
+				er.logger.Criticalw("TxAttempt has been uncofirmed for more than: ", er.config.EthTxResendAfterThreshold()*2,
```

Comment:
Hmm, I think a better category would be Error, not Critical.
Critical is for cases when a NOP should seriously look into their node for config errors etc.

Reply:
I think Critical is more appropriate here because we are not sure whether the transaction has gotten stuck or not. There is a chance (depending on the delay) that it might eventually get resolved, but in Error's case, it's certain that something has gone wrong.
For example, we use critical when we've reached the gas bump limit but the transaction hasn't yet been confirmed which is an intermediate state.

Reply:
Please confirm with @samsondav or @jmank88 about the correct log level here.

Reply:
Hmmm originally we made critical level for things a node op can do something about, but I'm not sure that's the rule in practice. We also have a new `HealthReport() map[string]error` - even if the node op can't fix anything and this is only error level, should the resender report as unhealthy? cc @essamhassan